### PR TITLE
bbpanzu wanted this

### PR DESCRIPTION
**Pull request by fireredfan7**

### CODE REVIEW

File: UPDATE HAXE TO 4.2.4.txt [hunk 1]
```diff
@@ -6,4 +7,7 @@ UPDATE HAXE TO 4.2.4
 UPDATE HAXE TO 4.2.4
 UPDATE HAXE TO 4.2.4
 UPDATE HAXE TO 4.2.4
-UPDATE HAXE TO 4.2.4
\ No newline at end of file
+UPDATE HAXE TO 4.2.4
+
+fireredfan7:
```
Comment (tposejank) [inline UPDATE HAXE TO 4.2.4.txt:12]: Uneccesary change.

File: UPDATE HAXE TO 4.2.4.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+bbpanzu:
```
Comment (tposejank) [inline UPDATE HAXE TO 4.2.4.txt:1]: Uneccesary change.

**Review by tposejank — CHANGES_REQUESTED** [review]
Changing UPDATE HAXE TO 4.2.4 is uneccesary.

File: UPDATE HAXE TO 4.2.4.txt (hunk 2 above)
Comment (nintendofan44) [reply UPDATE HAXE TO 4.2.4.txt:1]: 4.2.0 is bad
this pr is not that bad

**Comment by Doritojoey** [comment]
There's no point to this

**Comment by fireredfan7** [comment]
I see

File: UPDATE HAXE TO 4.2.4.txt (hunk 2 above)
Comment (fireredfan7) [reply UPDATE HAXE TO 4.2.4.txt:1]: > Changing UPDATE HAXE TO 4.2.4 is uneccesary.

Your'e right.

**Pull request closed without merging**